docs/tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_realtime_unsupported.md

```diff
@@ -0,0 +1,52 @@
+# Test basic usage realtime unsupported
+
+The test check for the correct transition to FIM mode scheduled when realtime is not avaible in the current OS and is
+set in the ossec.conf as a monitorization mode.
```

Comment:
```suggestion
The test checks for FIM to properly change from realtime to scheduled mode when it is not supported.
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/a46ae8a444c0c0a7f68436a4ccee7af9780437c6)